.github/workflows/clear-runs.yml

```diff
@@ -0,0 +1,87 @@
+name: Clear outdated runs
+
+on:
+  workflow_call:
+    secrets:
+      github-token:
+        description: The token needed for the reject outdated runs endpoint.
+        required: true
+
+jobs:
+  get-previous-runs:
+    name: Get previous runs
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get CI/CD workflow
+        id: get-cicd-workflow
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const response = await github.rest.actions.listRepoWorkflows({
+               owner: context.repo.owner,
+               repo: context.repo.repo
+            })
+            console.log("Get CI/CD workflow response = ", response.data);
+            const cicdWorkflows = response.data.workflows.filter((workflow) => workflow.name === "CI/CD");
```

Comment:
Can we not keep this generic and match on the name/ref or the current workflow?

This way is both fragile and doesn't allow this workflow to be truly reusable.